STM32Resources/install.sh

```diff
@@ -6,7 +6,27 @@ if [[ $UID != 0 ]]; then
     exit 1
 fi
 
-apt-get -y install build-essential git libsane:i386 ia32-libs-multiarch autoconf libusb-1.0-0-dev lib32ncurses5 libncurses5:i386 software-properties-common pkg-config cmake
+# Determine what distribution of Linux the user has
+DISTRO = $(. /etc/os-release; echo $NAME)
```

Comment:
Does this work? I think bash script variables can't have spaces around the equal sign. Regardless, that's the norm so it should be written that way :)

Reply:
Whoops, I pushed the wrong file haha. I'll update the if statements and push again.